recipes/weeder/build.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+TGT="$PREFIX/share/weeder2"
+[ -d "$TGT" ] || mkdir -p "$TGT"
+[ -d "${PREFIX}/bin" ] || mkdir -p "${PREFIX}/bin"
+
+cd "${SRC_DIR}"
+g++ weeder2.cpp -o weeder2 -O3
+cp -r FreqFiles $TGT/
```

Comment:
The files are included in the source download, but this would be with the aim of reducing the size of the conda package, right?

I don't understand the relevance of your example. Do you mean adding a `post-link.sh`, like https://github.com/bioconda/bioconda-recipes/blob/master/recipes/krona/post-link.sh?